### DISCUSSION TITLE
Replacement to MA_ natives

### DISCUSSION THREAD

**Discussion by naydef** [Batfoxkid/Freak-Fortress-2-Rewrite#152] [Q&A] [answered]
There's a plugin that relies on the `MA_` group of native, created by `ff2bat_menu_abilities`.

Is there a replacement (or another method) for them when using `ff2r_default_abilities`?

The plugin in question uses:
```
MA_GetManaFloat
MA_GetBool
MA_GetBossBool
```

**Comment by Batfoxkid** [comment]
FF2R plugins focus on the ability to change settings using the boss config to modify ability arguments on the fly.

**Reply by naydef** [reply]
Ok, maybe i can see code example of changing mana amount by certain amount.

**Reply by Batfoxkid** [reply, marked as answer]
```kv
"character"
{
  "special_menu_manager"
  {
    "manas"
    {
      "11"
      {
        "maximum" "500.0"
      }
    }
  }
}
```

```sourcepawn
ConfigData mana = FF2R_GetBossData(client).GetAbility("special_menu_manager.manas.11");
mana.SetFloat("maximum", 1000.0);
```

**Reply by naydef** [reply]
Thanks
